sql/001_create_small_business_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `small_businesses` (
+  `id` int NOT NULL,
+  `name` varchar(45) NOT NULL,
+  `city` varchar(45) NOT NULL,
+  `zip` int NOT NULL,
+  `cause area` varchar(45) NOT NULL,
```

Comment:
I suggest being more consistent in field naming - having a space in a field name is technically acceptable, but in practice it creates issues for fellow developers - they have to remember that this field have to be escaped.

Please rename it to `cause_area`